core/src/main/resources/application.yml

```diff
@@ -846,6 +846,8 @@ existing-stack-patcher:
   intervalInHours: 24
   maxInitialStartDelayInHours: 12
   patch-configs:
+    mock:
```

Comment:
Do we really want to have the mock config in the production config? Isn't it activate a mock path on prod environment as well? Don't we have an integration test specific application config where we should place this config?

Reply:
I would like to run these on mow-* envs too, to collect and test EDH reports. But maybe there is a better place to enable the mock stack patch.

Reply:
This config is not even needed because of the special handling - starting the mock patch after cluster creation, so removed the config